src/nvme/api-types.h

```diff
@@ -677,7 +675,7 @@ struct nvme_copy_args {
 	int fd;
 	__u32 timeout;
 	__u32 nsid;
-	__u32 ilbrt;
+	__u64 ilbrt;
```

Comment:
This breaks the API

Reply:
I have one more patch for nvme-cli to fix this issue. In that patch, this renamed ilbrt variable is handled. I hope the API will not break once it is merged.

Reply:
As far I can tell, libnvme is not just used by nvme-cli. This is why I keep repeating we can't break the API for the 1.x series of the library. For 2.x sure, I don't have any problems to modify the API. 

BTW, I would suggest to open an in issue with the exact work description and queue it up for the 2.x library. I fear if we write it down, this will be lost.

Reply:
Hi @igaw 
I missed to mention a point.
In `struct nvme_copy_args`, `ilbrt` and `ilbrt_u64` are used to store the reference tag values of 32-bit and 64-bit respectively. The `ilbrt_u64` is only assigned with `cfg.ilbrt` value in nvme-cli. Whereas `ilbrt` is not assigned with any value. But, both the variables are handled in `nvme_copy()` libnvme. It seems fishy for me. That's why I tried to use single `ilbrt` instead of two variable with different data type.